main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidator.java

```diff
@@ -96,26 +96,26 @@ public void validate(NoticeContainer noticeContainer) {
   @GtfsValidationNotice(severity = ERROR, files = @FileRefs(GtfsShapeSchema.class))
   static class DecreasingShapeDistanceNotice extends ValidationNotice {
 
-    // The id of the faulty shape.
+    /** The id of the faulty shape. */
     private final String shapeId;
 
-    // The row number from `shapes.txt`.
+    /** The row number from `shapes.txt`. */
     private final int csvRowNumber;
 
-    // Actual distance traveled along the shape from the first shape point to the faulty record.
+    /** Actual distance traveled along the shape from the first shape point to the faulty record. */
     private final double shapeDistTraveled;
 
-    // The faulty record's `shapes.shape_pt_sequence`.
+    /** The faulty record's `shapes.shape_pt_sequence`. */
     private final int shapePtSequence;
 
-    // The row number from `shapes.txt` of the previous shape point.
+    /** The row number from `shapes.txt` of the previous shape point. */
     private final long prevCsvRowNumber;
 
     // Actual distance traveled along the shape from the first shape point to the previous shape
-    // point.
+    /** point. */
```

Comment:
```suggestion
    /** Actual distance traveled along the shape from the first shape point to the previous shape point. */
```

**Maybe it is too long for the formatting rule